lib/kafo/kafo_configure.rb

```diff
@@ -477,13 +481,7 @@ def run_installation
         log_parser = PuppetLogParser.new
         logger = Logger.new('configure')
 
-        start_message = <<-HEREDOC
-Starting system configuration.
-  The total number of configuration tasks may increase during the run.
```

Comment:
I just noticed that we no longer print this but it was something that surprised me the first time I saw it. Does it still make sense to warn the user about this?

Reply:
Usually, the increase in the total number of configuration tasks is small relative to the original reported total (less than 1% delta in my testing). I reason that because the delta is small, and because the concept is intuitive enough that one configuration might trigger another previously unplanned item, we could save the line of output.

If we want to make this clearer, I think we can do so without causing additional lines of output, with language like

`2021-03-12 15:11:10 [NOTICE] [configure] 250 configuration steps out of 2188 currently planned steps complete.`

Still, it reads a little bit clunky to me, and probably unnecessary?